### ISSUE TITLE
Editor: Texture map doesn't render when refresh web page in Chrome, but this problem doesn't reappear in Firefox.

### ISSUE THREAD

**Issue by linbingquan** [mrdoob/three.js#15180]
##### Three.js version

- [x] Dev
- [x] r97
- [ ] ...

##### Browser

- [ ] All of them
- [x] Chrome 70
- [ ] Firefox 63

##### OS

- [x] Windows

---

refresh page and mouse hover model before:
![image](https://user-images.githubusercontent.com/23699926/47775536-e9762880-dd2a-11e8-98d2-cb04940b04ec.png)
refresh page and mouse hover model after:
![image](https://user-images.githubusercontent.com/23699926/47775557-f4c95400-dd2a-11e8-87f8-6c9d87a8d3b0.png)

**Comment by Mugen87** [comment]
I can confirm this bug. The texture does render as soon as you transform the camera with the controls. There is also the following warning logged:
```
THREE.WebGLRenderer: Texture marked for update but image is incomplete
```

**Comment by elhigu** [comment]
I also started having the same problem with my old code, which is not waiting that video element is ready before starting to draw scene.

Works on firefox, but prints console errors on latest Chrome/Canary

``` 
WebGL: INVALID_VALUE: tex(Sub)Image2D: video visible size is empty
```

https://codepen.io/elhigu/pen/pozNRNa

Documentation doesn't say anything that user should wait for video to be loaded before starting render loop. https://threejs.org/docs/#api/en/textures/VideoTexture

So is this a bug in library or missing documentation or am I just doing something wrong?

**Comment by elhigu** [comment]
Or should I maybe open a separate issue for this?

**Comment by Mugen87** [comment]
It seems your issue has nothing to do with `three.js`. At least on Chrome you can't just load and play a video with sound. This violates a security policy. As demonstrated in the [example](https://threejs.org/examples/webgl_materials_video), you have to start the video after a user interaction. Or you just mute the video like demonstrated in this fiddle:

https://jsfiddle.net/ugm03821/

**Comment by elhigu** [comment]
@Mugen87 thanks! That seems to be the reason why it doesn't load. Strangely it seems to work every now and then even when video element is started with autoplay and not being muted.

Is there a way to initialize video texture in a way that playback in not started? If I create element without autoplay in a way that first frame is loaded anyways? just calling videoElement.load() didn't seem to work.

Right now only workaround I have found is to initialize the video like this:

```
  const videoElement = document.createElement('video');
  videoElement.src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/175382/SampleVideo_1280x720_1mb.mp4";
  videoElement.crossOrigin = 'anonymous';
  videoElement.loop = true;
  videoElement.muted = true;
  videoElement.play().then(() => {
    videoElement.pause();
    videoElement.muted = false;
  });
```
In our use case we actually need should just load video texture that is not playing, but is still rendered.

**Comment by elhigu** [comment]
Also this seems to be able to "autoplay" video without user interaction in chrome:

```
  videoElement.play().then(() => {
    videoElement.pause();
    videoElement.muted = false;
  }).then(() => videoElement.play());
```

Chrome bug maybe if unmuted autoplay should not work without user interaction? 🤔 I updated https://codepen.io/elhigu/pen/pozNRNa with unmuted autoplay workaround.

**Comment by Mugen87** [comment]
> Is there a way to initialize video texture in a way that playback in not started?

I would say just don't set `autoplay` to `true` and avoid to call `videoElement.play()`.

**Comment by elhigu** [comment]
> I would say just don't set `autoplay` to `true` and avoid to call `videoElement.play()`.

That way I get the `WebGL: INVALID_VALUE: tex(Sub)Image2D: video visible size is empty` error 😥 to console.

https://codepen.io/elhigu/pen/OJLbgdj?editors=0010#0

**Comment by Mugen87** [comment]
If you want avoid the warning, I suggest you create the texture when you actually want to play the video.

**Comment by elhigu** [comment]
> If you want avoid the warning, I suggest you create the texture when you actually want to playback the video.

Well... in that case I wouldn't be able to show first frame of stopped video before starting playback.

**Comment by Mugen87** [comment]
> in that case I wouldn't be able to show first frame of stopped video before starting playback.

I was not aware that you want to show the first frame, sry.
